src/Stepper/Step.js

```diff
@@ -0,0 +1,180 @@
+      step: {
+        width: `${width}%`,
+        display: 'table-cell',
+        position: 'relative',
+        paddingTop: circleTop,
+      },
+
+      circle: {
+        width: size,
+        height: size,
+        margin: '0 auto',
+        backgroundColor: 'transparent',
+        borderRadius: '50%',
+        borderWidth: 3,
+        borderStyle: 'solid',
+        borderColor: '#fff',
+        textAlign: 'center',
+        padding:1,
+        fontSize: circleFontSize,
+        color: circleFontColor,
+        display: 'block',
+        opacity: defaultOpacity,
+
+      },
+      activeCircle: {
+        backgroundColor: activeColor,
+        opacity: activeOpacity
+      },
+      completedCircle: {
+        backgroundColor: completeColor,
+        opacity: completeOpactiy,
+      },
+      index: {
+        lineHeight: `${size + circleFontSize / 4}px`,
+        color: circleFontColor
+      },
+      title: {
+        marginTop: titleTop,
+        fontSize: titleFontSize,
+        fontWeight: '300',
+        textAlign: 'center',
+        display: 'block',
+        color: defaultTitleColor,
+        opacity: defaultOpacity
+      },
+      activeTitle: {
+        color: activeTitleColor,
+        opacity: activeOpacity
+      },
+      completedTitle: {
+        color: completeTitleColor,
+        opacity: completeOpactiy
+      },
+      leftBar: {
+        position: 'absolute',
+        top: circleTop + size / 2,
+        height: 1,
+        borderTopStyle: 'dashed',
+        borderTopWidth: 1,
+        borderTopColor: defaultColor,
+        left: 0,
+        right: '50%',
+        marginRight: size / 2 + 5,
+        opacity: defaultOpacity
+      },
+      rightBar: {
+        position: 'absolute',
+        top: circleTop + size / 2,
+        height: 1,
+        borderTopStyle: 'dashed',
+        borderTopWidth: 1,
+        borderTopColor: defaultColor,
+        right: 0,
+        left: '50%',
+        marginLeft: size / 2 + 5,
+        opacity: defaultOpacity
+      },
+      completedBar: {
+        borderTopStyle: 'dashed',
+        borderTopWidth: 1,
+        borderTopColor: completeColor,
+        opacity: completeOpactiy
+      },
+    };
+  }
+
+  render() {
+    const { title, index, active, completed, first, isLast, href, onClick } = this.props;
+
+    const styles = this.getStyles();
+    const circleStyle = Object.assign(
+      styles.circle,
+      completed ? styles.completedCircle : {},
+      active ? styles.activeCircle : {},
+    );
+
+    const titleStyle = Object.assign(
+      styles.title,
+      completed ? styles.completedTitle : {},
+      active ? styles.activeTitle : {},
+    );
+    const leftStyle = Object.assign(styles.leftBar, (active || completed) ? styles.completedBar : {});
+    const rightStyle = Object.assign(styles.rightBar, completed ? styles.completedBar : {});
+
+    return (
+      <div style={ styles.step }>
+          <div style={ circleStyle }>
+            {active || completed ? (
```

Comment:
So if I change the forked version of the react-stepper repo on trabian's profile can we pull that into the project and use it?